### PR TITLE
added mysql settings to settings.py

### PR DESCRIPTION
**Pull request by SoumyaRanjanNaik**
added settings for the MySQL in settings.py.
Details that need to be entered or changed are marked with TODOs.

This resolves the issue #31 .

### CODE REVIEW

File: rtask/rtask/settings.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'qgw%q&l*_*^pa5r&%82cz4&-71!lhhfe3ujc!z(j(y27@50iiz'
+SECRET_KEY = "qgw%q&l*_*^pa5r&%82cz4&-71!lhhfe3ujc!z(j(y27@50iiz"
```
Comment (m-henderson) [inline rtask/rtask/settings.py:23]: why was this changed?

**Review by m-henderson — CHANGES_REQUESTED** [review]
Thanks for the PR! could you please remove the unnecessary changes where you changed ' to "?

**Comment by SoumyaRanjanNaik** [comment]
Sorry, my mistake I use black as my formatter which did this, I will fix it give me some time.

**Comment by SoumyaRanjanNaik** [comment]
@m-henderson please check now this should be according to your requirements.

**Review by m-henderson — CHANGES_REQUESTED** [review]
looks good and is almost ready to merge. Could you please remove that "if DEBUG"  comment from lines 76 to 83?

**Comment by SoumyaRanjanNaik** [comment]
> looks good and is almost ready to merge. Could you please remove that "if DEBUG" comment from lines 76 to 83?

Actually that line will help you to keep proper separation between your development and production environments by switching between databases based on the DEBUG value.

May I uncomment the condition so you can see it working ?

**Comment by m-henderson** [comment]
> > looks good and is almost ready to merge. Could you please remove that "if DEBUG" comment from lines 76 to 83?
> 
> Actually that line will help you to keep proper separation between your development and production environments by switching between databases based on the DEBUG value.
> 
> May I uncomment the condition so you can see it working ?

This is not the best way to manage different environment variables. For example, it's better to create a different settings file for dev, qa, prod, that contains the values that should be overridden. Then, the correct settings will be run for the corresponding environment the app is in. So, If I am in dev then dev_settings.py should run and override only certain values in settings.py. I will remove the comment because I am pulling down the code to test it anyway :)

**Comment by m-henderson** [comment]
@SoumyaRanjanNaik actually, I just tried to make the changes and it is being a pain. Could you make the changes? I will leave comments in the file.

File: rtask/rtask/settings.py [hunk 2]
```diff
@@ -31,52 +31,67 @@
...
 ]
 
-WSGI_APPLICATION = 'rtask.wsgi.application'
+WSGI_APPLICATION = "rtask.wsgi.application"
 
 
 # Database
 # https://docs.djangoproject.com/en/3.1/ref/settings/#databases
 
+# if DEBUG:
```
Comment (m-henderson) [inline rtask/rtask/settings.py:76]: please remove this if else comment block

File: rtask/rtask/settings.py [hunk 3]
```diff
@@ -31,52 +31,67 @@
...
+#         }
+#     }
+# else:
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+    "default": {
+        "ENGINE": "django.db.backends.mysql",
+        "NAME": "",  # TODO: Database Name
```
Comment (m-henderson) [inline rtask/rtask/settings.py:87]: please remove this TODO comment. There is nothing to do here.

File: rtask/rtask/settings.py [hunk 4]
```diff
@@ -31,52 +31,67 @@
...
+        "USER": "",  # TODO: username for the database
```
Comment (m-henderson) [inline rtask/rtask/settings.py:88]: please remove this TODO comment. There is nothing to do here.

File: rtask/rtask/settings.py [hunk 5]
```diff
@@ -31,52 +31,67 @@
...
+        "PASSWORD": "",  # TODO : Password for Database
```
Comment (m-henderson) [inline rtask/rtask/settings.py:89]: please remove this TODO comment. There is nothing to do here.

File: rtask/rtask/settings.py [hunk 6]
```diff
@@ -31,52 +31,67 @@
...
+        "HOST": "localhost",  # TODO: replace this with hostname in production
```
Comment (m-henderson) [inline rtask/rtask/settings.py:90]: please remove this TODO comment. There is nothing to do here.

**Comment by SoumyaRanjanNaik** [comment]
okay sure.

**Review by m-henderson — CHANGES_REQUESTED** [review]
ok. get those comments fixed up and it will be ready to merge.

File: rtask/rtask/settings.py [hunk 7]
```diff
@@ -73,21 +73,13 @@
...
-#     }
-# else:
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.mysql",
-        "NAME": "",  # TODO: Database Name
-        "USER": "",  # TODO: username for the database
-        "PASSWORD": "",  # TODO : Password for Database
-        "HOST": "localhost",  # TODO: replace this with hostname in production
+        "NAME": "",  # Database Name
```
Comment (m-henderson) [inline rtask/rtask/settings.py:79]: no need for comments. we know that is the database name, username, password, and host.

**Comment by m-henderson** [comment]
@SoumyaRanjanNaik take care of that one last comment and your good.

**Review by m-henderson — APPROVED** [review]

**Pull request merged**